packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,212 @@
+    {slots.presence && <slots.presence {...slotProps.presence} />}
+    {slots.primaryText && <slots.primaryText {...slotProps.primaryText} />}
+    {slots.secondaryText && <slots.secondaryText {...slotProps.secondaryText} />}
+    {slots.tertiaryText && <slots.tertiaryText {...slotProps.tertiaryText} />}
+    {slots.quaternaryText && <slots.quaternaryText {...slotProps.quaternaryText} />}
+  </slots.root>
+);
+```
+
+- _**DOM**_
+
+```html
+<div class="fui-Persona">
+  <div {/* Avatar, PresenceBadge, or Avatar with PresenceBadge */} />
+  <span class="fui-Persona__primaryText">Primary Text</span>
+  <span class="fui-Persona__secondaryText">Secondary Text</span>
+  <span class="fui-Persona__tertiaryText">Tertiary Text</span>
+  <span class="fui-Persona__quaternaryText">Quaternary Text</span>
+</div>
+```
 
-_Describe what will need to be done to upgrade from the existing implementations:_
+## Migration
 
-- _Migration from v8_
-- _Migration from v0_
+See [MIGRATION.md](./MIGRATION.md) for details.
 
 ## Behaviors
 
 _Explain how the component will behave in use, including:_
 
 - _Component States_
+  - There are no states for this component, it's a visual representation.
 - _Interaction_
   - _Keyboard_
+    - Doesn't receive focus.
   - _Cursor_
+    - Doesn't interact with cursor.
   - _Touch_
+    - Doesn't interact with touch.
   - _Screen readers_
+    - It first focuses on the media and then goes through each text line available.
 
 ## Accessibility
 
-Base accessibility information is included in the design document. After the spec is filled and review, outcomes from it need to be communicated to design and incorporated in the design document.
-
-- Decide whether to use **native element** or folow **ARIA** and provide reasons
-- Identify the **[ARIA](https://www.w3.org/TR/wai-aria-practices-1.2/) pattern** and, if the component is listed there, follow its specification as possible.
-- Identify accessibility **variants**, the `role` ([ARIA roles](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)) of the component, its `slots` and `aria-*` props.
-- Describe the **keyboard navigation**: Tab Oder and Arrow Key Navigation. Describe any other keyboard **shortcuts** used
-- Specify texts for **state change announcements** - [ARIA live regions
-  ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) (number of available items in dropdown, error messages, confirmations, ...)
-- Identify UI parts that appear on **hover or focus** and specify keyboard and screen reader interaction with them
-- List cases when **focus** needs to be **trapped** in sections of the UI (for dialogs and popups or for hierarchical navigation)
-- List cases when **focus** needs to be **moved programatically** (if parts of the UI are appearing/disappearing or other cases)
+- There's no need for `aria-*` or/and`role`. Avatar and Badge already accessible and the text labels won't need anything as well.
```

Comment:
We can follow up on this later, but the Avatar's `aria-label` is the `name` prop. If that's also repeated in `primaryText`, then that name would be read out twice. Need to check that and figure out the best approach (perhaps add `aria-hidden` on the Avatar, but should definitely check into what's the right solution).

Reply:
I just checked with Sarah offline, there's no correct answer to this. I checked what owa and teams are doing, and they do not hide the first line. I think we could leave as is and based on partner feedback we can refine it. One of the main reasons why I think it should stay as is, because we would have to hide it only when an Avatar is used since we can have a PresenceBadge alone. I am open to doing this if you prefer, but I do think that maybe following what we currently have is okay.

Reply:
To clarify, I was suggesting possibly hiding the Avatar, not the text. We definitely shouldn't hide the text :)